src/NServiceBus.Core/Serializers/Json/MessageContractResolver.cs

```diff
@@ -9,7 +9,6 @@ class MessageContractResolver : DefaultContractResolver
         IMessageMapper messageMapper;
 
         public MessageContractResolver(IMessageMapper messageMapper)
-            : base(true)
```

Comment:
Was obsoleted in json.net. The caching is no longer part of the lib. It is the responsibility of the user of the lib